r/NEWS.md

```diff
@@ -50,6 +50,7 @@
 
 * Non-UTF-8 strings (common on Windows) are correctly coerced to UTF-8 when passing to Arrow memory and appropriately re-localized when converting to R
 * The `coerce_timestamps` option to `write_parquet()` is now correctly implemented.
+* The `version` option to `write_parquet()` is now correctly implemented.
```

Comment:
Please move this up to line 21 so that it's in the current (development) version.

Reply:
Changed. Thanks for your patience.